city/crud.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import Session
+
+from . import models
+from city import schemas
+
+
+def get_city_by_name(db: Session, name: str):

```

Comment:
Why do you need this function?

Reply:
This CRUD method I use in 'post endpoint' to check new city name in database
CRUD method 'get_city_by_id' I use only in 'get endpoint' to receive city by id parameter